<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uregister</class>
 <widget class="QDialog" name="uregister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DMS Client</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="namelabel">
        <property name="text">
         <string>Usernme </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="unameEdit">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="pwlabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="repwlabel">
        <property name="text">
         <string>Confirm   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="repwEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="phonelabel">
        <property name="text">
         <string>Phone       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneEdit">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="maillabel">
        <property name="text">
         <string>E-mail       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mailEdit">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="reset">
        <property name="text">
         <string>ClearOut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submit">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
